### PR TITLE
Added Contributing & templates

### PR DESCRIPTION
**Pull request by marbar3778**
- Contributing.md
- Templates issues, pull requests
Signed-off-by: Marko Baricevic <marbar3778@yahoo.com>

Would we want to add a whole contributing.md here or leave it as a reference to the tendermint one?

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+v    If a checkbox is n/a - please still include it but + a little note why
+☺ > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > >  -->
+
+- Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/tendermint/tendermint/blob/master/CONTRIBUTING.md))
+
+- [ ] Linked to github-issue with discussion and accepted design OR link to spec that describes this work.
+- [ ] Wrote tests
+- [ ] Updated relevant documentation (`docs/`)
+- [ ] Added a relevant changelog entry: `clog add [section] [stanza] [message]`
+- [ ] rereviewed `Files changed` in the github PR explorer
```
Comment (Liamsi) [inline .github/PULL_REQUEST_TEMPLATE.md]: Should we still add a line here that states sth like:
```suggestion
- [ ] rereviewed `Files changed` in the github PR explorer
- [ ] bumped version number of crate if this is meant to be released
- [ ] bumped supported version in README.md if tendermint version was updated
```
Also not sure if "rereviewed" shouldn't be just "reviewed"?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Contributing
+
+Thank you for taking the time to contribute to our organization and this repository. 
+
+Please [Reference Tendermint](https://github.com/tendermint/tendermint/blob/master/CONTRIBUTING.md) for the documentation on how to contribute.
```
Comment (Liamsi) [inline CONTRIBUTING.md]: The tendermint contributing guidline is a bit too complex for this repo. Maybe it is still a good idea to link to this but somehow stating that this just serves as a good general guidline although there are differences in the projects? Does that make any sense?

Comment (tomtau) [reply CONTRIBUTING.md]: Perhaps rephrasing this a bit better https://github.com/jTendermint/jabci/wiki/Contribute :)

and basic Rust instructions from
https://github.com/tendermint/kms#development ?

Comment (Liamsi) [reply CONTRIBUTING.md]: In the call I was against the dichotomy of develop and master but it seems like publishing the crate depends on pushing to master:
https://github.com/tendermint/rust-abci/blob/3f51194e80aecf5af520655637e84476af9cebd1/.circleci/config.yml#L37

I'll change the base branch of this PR to develop then.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (marbar3778) [reply .github/PULL_REQUEST_TEMPLATE.md]: does it still make sense to bump version in PR now, since we are doing master and develop branches?

Comment (Liamsi) [reply .github/PULL_REQUEST_TEMPLATE.md]: No, you are right.

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -6,7 +6,7 @@ This code of conduct applies to all projects run by the Tendermint/COSMOS team a
 
 
 # Conduct
-## Contact: adrian@tendermint.com
+## Contact: ismail@tendermint.com
```
Comment (marbar3778) [inline CODE_OF_CONDUCT.md]: @Liamsi let me know if this is okay? otherwise, I can put mine for the time being

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+All contributors are expected to follow (Code of Conduct)[./CODE_OF_CONDUCT.md]
+
+## Feature requests and bug reports
+--- 
+Feature requests or bug reports can be posted as a (Github issue)[https://github.com/tendermint/rust-abci/issues/new]. In the issue, please follow the template and describe to the best of your ability what you did, what you expected and what happened instead.
+
+## Working on Issues
+---
+
+There are several ways to identify an area where you can contribute to Chain:
```
Comment (tomtau) [inline CONTRIBUTING.md]: s/Chain/rust-abci ?

File: CODE_OF_CONDUCT.md (hunk 3 above)
Comment (Liamsi) [reply CODE_OF_CONDUCT.md]: That's OK.

**Comment by Liamsi** [comment]
I'm wondering how to proceed with the current disparity of develop and master here:
image

I propose to merge this PR to develop (instead of currently master) and a PR that merges master into develop ASAP.

**Pull request merged**